.github/workflows/spmd_tests.yaml

```diff
@@ -35,15 +35,16 @@ jobs:
           python -m pip install --upgrade pip
           pip install pytest-shard
           if [ -f spmd/requirements_dev.txt ]; then pip install -r spmd/requirements_dev.txt; fi
+          if [ -f spmd/requirements.txt ]; then pip install -r spmd/requirements.txt; fi
```

Comment:
Why can't we update `requirements_dev.txt` also?

Reply:
> Why can't we update `requirements_dev.txt` also?

In spmd_GPU_tests, we have a line for requirements.txt originally. https://github.com/pytorch/tau/blob/main/.github/workflows/spmd_gpu_tests.sh#L20 
So I am including both.